src/Samples/Sample.AspNetCore.SystemTests/Services/FluentExtensions.cs

```diff
@@ -38,14 +38,21 @@ public static TOwner StorePrice<TOwner>(this UIComponent<TOwner> component, out
             return component.Owner;
         }
 
-        public static TOwner StoreValue<TOwner>(this UIComponent<TOwner> component, out Uri value)
+        public static TOwner StoreUri<TOwner>(this UIComponent<TOwner> component, out Uri value)
             where TOwner : PageObject<TOwner>
         {
             var val = component.Content.Value;
             value = new Uri(val, UriKind.RelativeOrAbsolute);
             return component.Owner;
         }
 
+        public static TOwner StoreValue<TOwner>(this UIComponent<TOwner> component, out string value)
```

Comment:
Codacy found an issue: [Consider refactoring this method in order to remove the need for this 'out' modifier.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6074055)